tools/ldcontext_generator.py

```diff
@@ -1,23 +1,45 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 """
-Extracts the properties, types and  enumerations of a JSON Schema
-converting them into terms of a JSON-LD @Context
+This script provides two files:
+ - context.jsonld, that serves https://schema.lab.fiware.org/ld/fiware-datamodels-context.jsonld
+ - mapping_list.yml, that serves  https://uri.fiware.org/ns/datamodels
+
+context.jsonld is combined by extracting the properties, types and  enumerations of a JSON Schema and
+converting them into terms of a JSON-LD @Context. mapping_list.yml uses the result of extracting
+to prepare a list of terms with schemas and specifications.
 
 Copyright (c) 2019 FIWARE Foundation e.V.
 
 Author: José M. Cantera
 """
 
-import sys
 import json
+import yaml
 import os
 from datetime import datetime
+from argparse import ArgumentParser
 
 # Here the aggregated @context will be stored
 aggregated_context = {
 }
 
+# Here a list of mappings (terms->schemas/specifications) will be stored
+terms_list = {
+    "terms": {}
+}
+
```

Comment:
can you ellaborate what an alert is?

Reply:
Yes, if associated with the schema spec file is not exists, the script will inform about it

Reply:
please add this explanation to the comment ;)

Reply:
Fixed in 8933b1ef27836a637fd56c3530e5b226b30a2fdb